tests/config.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- hosts: all
+
+  vars:
+    manala_sensu_config:
+      EMBEDDED_RUBY: true
```

Comment:
I use `manala_sensu_config.EMBEDDED_RUBY` to define ruby gem executable. ([here](https://github.com/manala/ansible-role-sensu/blob/master/tasks/gems.yml#L4))

It's much more difficult to have the same result with our macro structure : 
```
manala_sensu_config:
  - EMBEDDED_RUBY: true
```

Maybe we can give a try to new [json_query filter](http://docs.ansible.com/ansible/playbooks_filters.html#json-query-filter)... 

I propose to keep this structure for the release of version 1.0.0